### ISSUE TITLE
[Question]: Uptrain existing model

### ISSUE THREAD

**Issue by GKI98** [flairNLP/flair#3116]
### Question

Hi, I'm new here. I've trained ner model with flair and it works just fine. However, I need to add another NER to existing model with my other custom ner entity. How could I do that? Unfortunately Flair doesn't produce config.json so I don't understand how to upload my .pt model to huggingface to get it for the training pipeline.

Or maybe it is not related to flair itself, but I still don't get how to do what I need :(

**Comment by helpmefindaname** [comment]
Hi @GKI98 
If I understand you right, you want to reuse the transformer embeddings, while training a new NER model with additional data?
You can do that like this:
```
old_model = SequenceTagger.load("model.pt")
new_corpus = ....

label_dict = new_corpus.make_label_dictionary("ner", add_unk=False)
model = SequenceTagger(embeddings=old_model.embeddings, tag_dictionary=label_dict, use_rnn=False, use_crf=False, reproject_embeddings=False)

and then train it the common way.
```

**Comment by GKI98** [comment]
Thank you for a quick response! 

Well, yes, I want to add another ner entity to my ner model. E.g. I have trained my model to recognize entity "street". Now I want to train this model to extract both 'street' and 'house' entities. 

Thus, full code looks like this:
Training process runs but the model is not training somehow. Loss in very small and F1=0 from 1 to 12 epochs.

```
from flair.models import SequenceTagger
from flair.trainers import ModelTrainer

import flair, torch
flair.device = torch.device('cuda') 

# define columns
columns = {0 : 'text', 1:'ner'} 
data_folder = 'e:\\GoshaNLP'
corpus: Corpus = ColumnCorpus(data_folder, columns,
                              train_file = 'train.txt',
                              test_file = 'test.txt',
                              dev_file = 'dev.txt')

corpus = corpus
print(corpus)

label_type = 'ner'

label_dict = corpus.make_label_dictionary(label_type=label_type)
print(label_dict)

old_model = SequenceTagger.load("e:\\GoshaNLP\model\model.pt")

model = SequenceTagger(embeddings=old_model.embeddings, tag_dictionary=label_dict, use_rnn=False, use_crf=False, reproject_embeddings=False,
                        tag_type='house')

trainer = ModelTrainer(model, corpus)

trainer.fine_tune('e:\\GoshaNLP',
                  learning_rate=5.0e-6,
                  mini_batch_size=2,
                  mini_batch_chunk_size=1,  # remove this parameter to speed up computation if you have a big GPU
                  max_epochs=2)
```

**Comment by helpmefindaname** [comment]
If only the training loss is low, it is indicating overfitting. If both training loss and validation loss are low, I would check if the labels are loaded correctly, as then something is fishy.
However, I hope you understand, that I cannot help you without fully understanding your datasets & training results. It might be that the LR is too hight or too low or something completely different.

**Comment by GKI98** [comment]
Ofc, I see. Thank you! I just needed to be sure that I put every piece right. Probably some problem with labeling on my side.
